app/src/main/java/com/sumin/vknewsclient/presentation/comment/ScreeenComments.kt

```diff
@@ -0,0 +1,132 @@
+package com.sumin.vknewsclient.presentation.comment
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.material.TopAppBar
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.ArrowBack
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.sumin.vknewsclient.domain.model.comment.PostComment
+import com.sumin.vknewsclient.domain.model.post.FeedPostModel
+import com.sumin.vknewsclient.presentation.ui.theme.VkNewsClientTheme
+
+@Composable
+fun ScreenComments(
+    onBackPressed: () -> Unit,
+    feedPost: FeedPostModel
+) {
+    val viewModel: CommentsViewModel = viewModel(factory = CommentsViewModelFactory(feedPost))
+
+    val state = viewModel.screenState.collectAsState(CommentsScreenState.Initial)
```

Comment:
можно сразу значение достать из стейта через by:

```suggestion
    val state by viewModel.screenState.collectAsState(CommentsScreenState.Initial)
```